src/components/pic-title-header/pic-title-header.js

```diff
@@ -11,7 +11,7 @@ const SocialBtn = ({icon, text, url, name}) => {
     if (name.endsWith('s')) return `${name}'`;
     return `${name}'s`;
   }, [name])
-  return <OutboundLink className='unlink baseBtn lowercase prependIcon' href={url}>
+  return <OutboundLink className='unlink baseBtn lowercase prependIcon' target="_blank" href={url} >
```

Comment:
```suggestion
  return <OutboundLink className='unlink baseBtn lowercase prependIcon' target="_blank" rel="noopener" href={url} >
```

[Read more on `noopener` for external links](https://developers.google.com/web/tools/lighthouse/audits/noopener)